creator/Documents/DifferencesBetweenBedrockAndJava.md

```diff
@@ -8,95 +8,133 @@ description: "An article covering the major differences between Minecraft: Bedro
 
 # Differences Between Minecraft: Bedrock Edition and Minecraft: Java Edition
 
-On the surface, Bedrock Edition and Java Edition seem very similar, but under the hood is a completely different story. The different code bases create distinct development environments. This tutorial outlines the major differences you as a content creator should be aware of.
+On the surface, Bedrock Edition and Java Edition may seem very similar, but under the hood is a completely different story. The different code bases create distinct development environments for content creators. This tutorial outlines the major differences you as a content creator should be aware of.
 
 In this tutorial you will learn the following:
 
 > [!div class="checklist"]
 >
-> - A brief history of Java Edition and Bedrock edition.
+> - A brief history of Java Edition and Bedrock Edition.
 > - How the two editions differ and what it means for content creation.
 
 There are two major versions of Minecraft.
 
 ### Minecraft: Java Edition
 
-This version was originally released in 2009. This version used to be called Minecraft until it was renamed to Minecraft: Java Edition in September 2017. As the name implies, it’s developed in Java and isn’t compatible with the current version of Minecraft for the most part. This edition is commonly referred to as simply ***Java***.
+This is the original version of Minecraft, first released in May 2009. In September 2017, coinciding with the launch of Bedrock Edition, the "Java Edition" subtitle was added to the title. As the name implies, this version of Minecraft is developed in the [Java programming language](https://en.wikipedia.org/wiki/Java_(programming_language)). It's available only on Windows, Mac, and Linux PCs. This edition is commonly referred to as simply ***Java***.
 
```

Comment:
Made clear that this is the original PC minecraft.
Made clear the platform limitations (since that's the most prominent difference between Java and Bedrock).
Removed "isn’t compatible with the current version of Minecraft" since how can it be incompatible with itself? Unless 'current version of Minecraft' means Bedrock in which case that line is egregious.
Is it ok to link to wikipedia? I do it a few times in here